stock_picking_accounting_date/tests/test_stock_picking_accounting_date.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2023 Quartile Limited
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import date, timedelta
+
+from odoo.addons.stock_account.tests.test_stockvaluation import TestStockValuation
+
+
+class TestStockValuation(TestStockValuation):
+    def setUp(self):
+        super(TestStockValuation, self).setUp()
+
+    def test_stock_picking_accounting_date(self):
+        self.product1.categ_id.property_cost_method = "fifo"
```

Comment:
```suggestion
```
I guess this assignment doesn't affect the test.  Can we ensure that the valuation is `real_time` instead?